[#1596] custom drag n drop label text

aluarius opened the pull request:
close: #1596 

## Description
Added optional props for custom drag'n'drop label text.

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

m0ksem reviewed (commented):
I think we also want to care about "Upload file" text here. Also, would be nice to have slots here, maybe. 

https://github.com/epicmaxco/vuestic-ui/blob/ec2c65ce510f285606f381573bbb63e4baa1c33e/packages/ui/src/components/va-file-upload/VaFileUpload.vue#L14-L25

aluarius commented:
> I think we also want to care about "Upload file" text here. Also, would be nice to have slots here, maybe.

Agree, slots looks fine here.

Derranion reviewed (approved):
Thanks,

Could you please add this to the docs description.

Because otherwise users will not know how to use it without checking source code.

[Here are some slots examples in our docs](https://vuestic.dev/en/ui-elements/select#slots).

m0ksem commented on packages/ui/src/components/va-file-upload/VaFileUpload.vue:
```diff
@@ -9,7 +9,7 @@
         v-if="dropzone"
       >
-        Drag’n’drop files or
+        <slot name="dropZoneLabel">Drag’n’drop files or</slot>
```
Having prop for label is more appropriate. We can use [Components Config]( https://vuestic.dev/en/services/components-config) for i18n for all components :) 

When I spoke about slot, I meant a slot that can be used to change overall file-upload visual (to replace default button and label text). Sorry for that. Let's better have props that can be changed in global config.

aluarius commented:
Added props for custom upload button & drop zone. Added slot for the whole upload area (no v-bind for events needed, cause there is hidden input, covering the whole area). Added examples and docs description.

m0ksem reviewed (approved):
Pretty nice now 😊
